CoorpAcademy/mobile
# Use flavors from version file instead of env var

## Detailed purpose of the PR

This PR lets use build flavor from version file instead of env var, as it's not used anymore.

### How to test it

Run `yarn start:e2e`
You should not see yellow warns banner

<img src="https://user-images.githubusercontent.com/8419208/78899200-54c23f00-7a75-11ea-880e-97bf7a0c1e02.png" width="200" />

### Related PR

https://github.com/CoorpAcademy/mobile/pull/488

### Testing strategy

- [ ] I ensured my code is tested
- [ ] I took care to mention breaking changes and extra libs if appropriate
